src/components/AdvancedSettingsAccordion.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  InputGroup,
+  InputRightElement,
+} from "@chakra-ui/react";
+import { useState } from "react";
+import { useFormContext } from "react-hook-form";
+
+import { getSmallestUnit, makeValidateDecimals } from "./SendFlow/utils";
+import colors from "../style/colors";
+import { mutezToTez, tezToMutez } from "../utils/format";
+import { Estimation, TEZ_DECIMALS } from "../utils/tezos";
+
+export const AdvancedSettingsAccordion = ({ index = 0 }) => {
+  const { register, getValues, setValue } = useFormContext<{
+    executeParams: Estimation[];
+  }>();
+
+  const [tezFee, setTezFee] = useState<string>(mutezToTez(getValues().executeParams[index].fee));
+
+  const handleFeeChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const newFeeValue = event.target.value;
+
+    if (makeValidateDecimals(TEZ_DECIMALS)(newFeeValue) === true) {
+      setTezFee(newFeeValue);
+      setValue(`executeParams.${index}.fee`, tezToMutez(newFeeValue) as number);
```

Comment:
if newFeeValue is not a `NaN` `tezToMutez` will return a bignumber, we need to call a `toNumber` function here